fury/molecular.py

```
@@ -0,0 +1,698 @@
+    methods to access atomic properties. It provides access to essential
+    functionality available in ``vtkPeriodicTable``. An object of this class
+    provides access to atomic information sourced from Blue Obelisk Data
+    Repository.
+    """
+
+    def atomic_symbol(self, atomic_number):
+        """Given an atomic number, returns the symbol associated with the
+        element.
+
+        Parameters
+        ----------
+        atomic_number : int
+            Atomic number of the element whose symbol is to be obtained.
+        """
+        return self.GetSymbol(atomic_number)
+
+    def element_name(self, atomic_number):
+        """Given an atomic number, returns the name of the element.
+
+        Parameters
+        ----------
+        atomic_number : int
+            Atomic number of the element whose name is to be obtained.
+        """
+        return self.GetElementName(atomic_number)
+
+    def atomic_number(self, element_name):
+        """Given a case-insensitive string that contains the symbol or name of
+        an element, return the corresponding atomic number.
+
+        Parameters
+        ----------
+        element_name : string
+            Name of the element whose atomic number is to be obtained.
+        """
+        return self.GetAtomicNumber(element_name)
+
+    def atomic_radius(self, atomic_number, radius_type):
+        """Given an atomic number, return either the covalent radius of the
+        atom or return the Van Der Waals radius of the atom depending on
+        radius_type.
+
+        Parameters
+        ----------
+        atomic_number : int
+            Atomic number of the element whose covalent radius is to be
+            obtained.
+        radius_type : string
+            Two valid choices -
+            * 'VDW' : for Van Der Waals radius of the atom
+            * 'Covalent' : for covalent radius of the atom
+        """
+        if radius_type == 'VDW':
```

Do you know which one ('VDW' or 'Covalent') is by default? 
You can raise an error if radius_type is something else.

---

Post comment, I made it such that it's 'VDW' by default and a value error is raised if it's something other than these two. Thanks!